omod/src/main/java/org/openmrs/module/patientflag_rest/web/resource/FlagsResource.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.patientflag_rest.web.resource;
+
+import java.util.List;
+
+import org.openmrs.Patient;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.patientflag_rest.util.FlagUtil;
+import org.openmrs.module.patientflag_rest.web.wrapper.FlagWrapper;
+import org.openmrs.module.patientflags.api.FlagService;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.annotation.Resource;
+import org.openmrs.module.webservices.rest.web.representation.DefaultRepresentation;
+import org.openmrs.module.webservices.rest.web.representation.FullRepresentation;
+import org.openmrs.module.webservices.rest.web.representation.Representation;
+import org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingReadableResource;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
+
+/**
+ * {@link Resource} for Patient Flag
+ *
+ * @author Stélio Moiane
+ *
+ */
+@Resource(name = RestConstants.VERSION_1
+		+ "/flags", order = 2, supportedClass = FlagWrapper.class, supportedOpenmrsVersions = { "1.8.*", "1.9.*",
+				"1.10.*", "1.11.*", "1.12.*" })
+// order must be greater than that for PatientResource(order=0) RESTWS-273
+public class FlagsResource extends BaseDelegatingReadableResource<FlagWrapper> {

```

Comment:
I just decided to leave this as is for now. Probably in the future; we'll rename these resources and and versions for resources.